plugins/finder/jevents/jevents.php

```diff
@@ -233,39 +191,38 @@ protected function index(FinderIndexerResult $item, $format = 'html')
 		// Initialize the item parameters.
 		$registry = new JRegistry;
 		$registry->loadString($item->params);
+
 		$item->params = JComponentHelper::getParams('com_jevents', true);
 		$item->params->merge($registry);
 
 		$registry = new JRegistry;
 		$registry->loadString($item->metadata);
 		$item->metadata = $registry;
 
-		// Trigger the onContentPrepare event.
-		$item->summary = FinderIndexerHelper::prepareContent($item->summary, $item->params);
-		$item->body = FinderIndexerHelper::prepareContent($item->body, $item->params);
-
-		// Build the necessary route and path information.
-		$itemid= $this->params->get("target_itemid",0);
-		$item->url = "index.php?option=com_jevents&task=icalevent.detail&evid=".$item->eventid."&Itemid=".$itemid;//$this->getURL($item->id, $this->extension, $this->layout);
-		$item->route = "index.php?option=com_jevents&task=icalevent.detail&evid=".$item->eventid."&Itemid=".$itemid;
+		// add custom field keyword
```

Comment:
This is something that is customized for our needs since I created our version of jevents finder plugin. Not everybody creates a custom field so this can be deleted from the finder plugin. - Jackson